packages/components/src/components/divider/divider.stories.ts

```diff
@@ -0,0 +1,93 @@
+import '../../solid-components';
+import { html } from 'lit';
+import { storybookDefaults, storybookTemplate } from '../../../scripts/storybook/helper';
+const { args, argTypes, parameters } = storybookDefaults('sd-divider');
+const { generateTemplate } = storybookTemplate('sd-divider');
+
+export default {
+  title: 'Components/sd-divider',
+  component: 'sd-divider',
+  args,
+
+  argTypes,
+
+  parameters: { ...parameters },
+  decorators: [
+    (story: () => typeof html) => html`
+      <style>
+        sd-divider[orientation='vertical'] {
+          height: 120px;
+        }
+        sd-divider[orientation='horizontal'] {
+          width: 120px;
+        }
```

Comment:
Please add a story which shows, how the size can influenced by just setting e. g. the width.

Show 2 or 3 sizes and maybe just do it with a template. (Don't do the %TEMPLATE% thing in the constants but just set different values on the y Axis – I believe that should work.)

Reply:
@mariohamann I added the sizes story but there might be a cleaner way to organise the CSS. Tried organising the story into x and y axis, but I couldn't figure out how to make variations based on orientation and the class names(since that is not a prop and was only needed in the class).